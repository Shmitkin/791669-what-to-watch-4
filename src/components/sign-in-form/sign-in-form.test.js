import React from "react";
import renderer from "react-test-renderer";

import SignInForm from "./sign-in-form.jsx";


it(`Should SignInForm render correctly`, () => {

  const tree = renderer
    .create(
        <SignInForm
          onSubmit={()=>{}}
          onChange={()=>{}}
          onFocus={()=>{}}
          errorMessage={`error message`}
          emailValidity={true}
          passValidity={true}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
