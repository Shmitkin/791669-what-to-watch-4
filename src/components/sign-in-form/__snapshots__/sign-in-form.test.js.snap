// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should SignInForm render correctly 1`] = `
<form
  className="sign-in__form"
  noValidate={true}
  onChange={[Function]}
  onSubmit={[Function]}
>
  <div
    className="sign-in__message"
  >
    <p>
      error message
    </p>
  </div>
  <div
    className="sign-in__fields"
  >
    <div
      className="sign-in__field "
    >
      <input
        className="sign-in__input"
        id="user-email"
        name="user-email"
        onFocus={[Function]}
        placeholder="Email address"
        type="email"
      />
      <label
        className="sign-in__label visually-hidden"
        htmlFor="user-email"
      >
        Email address
      </label>
    </div>
    <div
      className="sign-in__field "
    >
      <input
        className="sign-in__input"
        id="user-password"
        name="user-password"
        onFocus={[Function]}
        placeholder="Password"
        type="password"
      />
      <label
        className="sign-in__label visually-hidden"
        htmlFor="user-password"
      >
        Password
      </label>
    </div>
  </div>
  <div
    className="sign-in__submit"
  >
    <button
      className="sign-in__btn"
      type="submit"
    >
      Sign in
    </button>
  </div>
</form>
`;
